doc/guides/writing-tests.md

```diff
@@ -388,3 +388,7 @@ will depend on what is being tested if this is required or not.
 [node.green]: http://node.green/
 [test fixture]: https://github.com/google/googletest/blob/master/googletest/docs/Primer.md#test-fixtures-using-the-same-data-configuration-for-multiple-tests
 [directory structure overview]: https://github.com/nodejs/node/blob/master/test/README.md#test-directories
+
+### Test Coverage
+
+Details on generating a test coverage report can be accessed [here].(https://github.com/nodejs/node/blob/master/doc/guides/contributing/pull-requests.md#test-coverage) 
```

Comment:
1. It seems this should go before the bottom references section.
2. The period between `[here]` and `(https://...)` [breaks link rendering](https://github.com/mithunsasidharan/node/blob/73f4f349bf5639a3c22ee87cf39be42d193753ea/doc/guides/writing-tests.md#test-coverage).

Reply:
3. And maybe it should be wrapped before `[here]` to avoid long line.